testing/mocks/resourceresolver-mock/src/main/java/org/apache/sling/testing/resourceresolver/MockResourceResolverFactory.java

```diff
@@ -23,15 +23,20 @@
 import java.util.Map;
 import java.util.Stack;
 
+import org.apache.jackrabbit.JcrConstants;
 import org.apache.sling.api.resource.LoginException;
 import org.apache.sling.api.resource.ResourceResolver;
 import org.apache.sling.api.resource.ResourceResolverFactory;
 import org.osgi.service.event.EventAdmin;
 
+import com.google.common.collect.ImmutableMap;
+
 /**
  * Simple resource resolver factory
  */
 public class MockResourceResolverFactory implements ResourceResolverFactory {
+    
+    private static final String JCR_ROOT="jcr:root";
```

Comment:
@rombert  thanks for noticing this. I checked the second link when I chose the name for it.